forge-gui/res/cardsfolder/s/signal_the_clans.txt

```diff
@@ -6,6 +6,6 @@ SVar:DBChoose:DB$ ChooseCard | Defined$ You | Amount$ 1 | AtRandom$ True | Choic
 SVar:DBChange:DB$ ChangeZone | Origin$ Library | Destination$ Hand | Defined$ ChosenCard | StackDescription$ None | SubAbility$ DBShuffle | ConditionCheckSVar$ X | ConditionSVarCompare$ EQ3
 SVar:DBShuffle:DB$ Shuffle | Defined$ You | SubAbility$ DBCleanup
 SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
-SVar:X:Count$DifferentCardNames_Creature.IsRemembered
+SVar:X:Remembered$DifferentCardNames_Creature
```

Comment:
I don't see why this would need to be different than _Scalpelexis_ anyway

Reply:
removed